### ISSUE TITLE
`official.vision` seems missing in `tf-models-nightly`

### ISSUE THREAD

**Issue by laszlo-luminous** [tensorflow/models#10379]
## 1. The entire URL of the file you are using

N/A

## 2. Describe the bug

I am unable to import the `official.vision` module from the nightly binary builds on pypi (https://pypi.org/project/tf-models-nightly/).

## 3. Steps to reproduce
```sh
conda create -n tmp -y python=3.8 && conda activate tmp
pip install tf-models-nightly==2.7.0.dev20211118
python -c "from official import vision"
```
yields
```
 Traceback (most recent call last):
 File "<string>", line 1, in <module>
ImportError: cannot import name 'vision' from 'official' (.../miniconda3/envs/tmp/lib/python3.8/site-packages/official/__init__.py)
```

## 4. Expected behavior
Be able to import `official.vision`.

## 5. Additional context

I tried older versions of `tf-models-nightly` and found that the last build that I can import `official.vision` from is `2.5.0.dev20211105`. Interestingly, installing `pip install tf-models-official==2.7.0` doesn't produce the same issue, even though that package is newer than `tf-models-nightly-2.5.0.dev20211105`.

There is no issue with importing other modules like `official.nlp`.

## 6. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 20.04 or macOS 11.2.3 (reproduced bug on both)
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): tf-nightly-2.8.0.dev20211118
- Python version: 3.8

**Comment by saberkun** [comment]
Hi, sorry about this. it is because the init.py was accidentally removed. (our internal build system does not rely on it)
I will fix it soon.

I am not sure why Python3 pip still need this init.py. Please share your knowledge if you know if with py3, the init.py can go away for package release.

**Comment by laszlo-luminous** [comment]
Thanks, this has indeed resolved the issue, `tf-models-nightly-2.7.0.dev20211119` works as expected. As far as I know, you need a `__init__.py` for every subdirectory that you want treat as a package, even in python3: https://docs.python.org/3/tutorial/modules.html#packages
